fenix/app/src/main/java/org/mozilla/fenix/browser/BrowserFragment.kt

```diff
@@ -216,6 +227,12 @@ class BrowserFragment : BaseBrowserFragment(), UserInteractionHandler {
             owner = this,
             view = view,
         )
+
+        view.toScope().launch {
+            reviewQualityCheckViewModel.isBottomSheetVisible.collectLatest {
+                reviewQualityCheck.setSelected(selected = it, notifyListener = false)
+            }
+        }
```

Comment:
this looks good. The bottom sheet currently covers this section though. Is it long term going to open and rest above the awesomebar? or is that still one of the TBD parts of the specs?

Reply:
👍 Yeah that's how I understand the feature would work. 

Seeing how little space there is already for a lot of content, I don't think we would end up pushing this above the awesome bar and create a customized bottom sheet experience